### PR TITLE
Search Bar

### PR DESCRIPTION
**Pull request by Dlonls**
- [DEMO LINK](https://Dlonls.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Dlonls.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```
Comment (valerii-derkach) [inline src/style.css:111]: no need default here

use classes as selectors
checklist p1

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+.inputSmall {
+  padding-left: 33px;
+  height: 42px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
```
Comment (valerii-derkach) [inline src/style.css:75]: checklist p2

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+.inputBig::placeholder {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
```
Comment (valerii-derkach) [inline src/style.css:77]: move font family to html tag, and use inheritance where it is needed

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
```
Comment (valerii-derkach) [inline src/style.css:110]: check font weight on focus, it should be bold

**Review by valerii-derkach — CHANGES_REQUESTED** [review]
checklist

File: src/style.css (hunk 1 above)
Comment (Dlonls) [reply src/style.css:111]: Зробив через класи, але коли прибираю default. Зникає box-shadow.

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**